### PR TITLE
Pullapprove updates

### PR DESCRIPTION
**Pull request by josephperrott**
See individual commits.

### CODE REVIEW

File: .pullapprove.yml [hunk 1]
```diff
@@ -118,96 +105,40 @@ overrides:
...
+    <<: *defaults
     conditions:
-      - *can-be-global-approved
-      - *can-be-global-docs-approved
-    reviews:
-      request: 0 # Do not request any reviews from the reviewer group
-      required: 1 # Require that all PRs have approval from at least one of the users in the group
-      author_value: 0 # The author of the PR cannot provide an approval for themself
-      reviewed_for: ignored # All reviews apply to this group whether noted via Reviewed-for or not
+      - author in ["renovate[bot]"]
```
Comment (devversion) [inline .pullapprove.yml:131]: Does this mean that both the caretaker + the corresponding groups are assigned for renovate PRs? e.g. dev-infra + the caretaker? Unless I miss anything both these groups would become active if renovate updates `/package.json`

**Review by devversion — APPROVED** [review]
LGTM, just one comment. Nice cleanup with the global approval anchors.

**Review by alxhub — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
@josephperrott This could use a rebase and is having problems with lint.

**Comment by devversion** [comment]
@josephperrott I've rebased this PR, but I still want to confirm that the behavior is expected (renovate group + actual groups)

File: .pullapprove.yml (hunk 1 above)
Comment (josephperrott) [reply .pullapprove.yml:130]: Yes, for now lets do it this way, and then in the future we can consider moving this to be just the caretaker.

**Comment by jessicajaniuk** [comment]
This PR has conflicts with `13.3.x`. Switching to `target: minor`. Please create a patch PR, if you want this to land in `13.3.x`.

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 8d85f24ef3dd8a34cd9e5c6f53ce967e889c2dec.

**Pull request closed without merging**